lib/stripe/util.rb

```diff
@@ -2,6 +2,8 @@
 
 module Stripe
   module Util
+    @invalid_params = Set.new [:idempotency_key, :api_key, :stripe_account]
```

Comment:
Lets make this a constant and give it a slightly more descriptive name, okay? `KNOWN_OPTS` or something like that.